src/main/java/core/basesyntax/Table.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Table {
+    private int buy;
+    private int supply;
+    private int result;
+
+    public Table(int buy, int supply) {
+        this.buy = buy;
+        this.supply = supply;
+        setResult(buy, supply);
+    }
+
+    public int getBuy() {
+        return buy;
+    }
+
+    public int getSupply() {
+        return supply;
+    }
+
+    public int getResult() {
+        return result;
+    }
+
+    private void setResult(int buy, int supply) {
+        this.result = supply - buy;
+    }
```

Comment:
If the 'result' field is necessary to be part of the object's state, consider renaming the method to something more descriptive, like 'calculateResult', to better reflect its purpose and follow the method naming convention.